runtime/src/iree/hal/local/local_executable_cache.c

```diff
@@ -15,6 +15,7 @@ typedef struct iree_hal_local_executable_cache_t {
   iree_hal_resource_t resource;
   iree_allocator_t host_allocator;
   iree_string_view_t identifier;
+  iree_host_size_t worker_capacity;
```

Comment:
It's not clear what the relationship is between the parts of this diff that refer to `worker_capacity` and the rest. Should that be a separate PR?

Reply:
Check the commit descriptions when reviewing PRs with multiple commits :)
In order to pass in per-worker state like the processor ID (as you asked for) per-worker state needed to be allocated and that requires knowing how many workers there are.